File: src/main/java/moe/giga/discord/commands/IdCommand.java
package moe.giga.discord.commands;

import moe.giga.discord.annotations.CommandInfo;
import moe.giga.discord.contexts.MessageContext;

@CommandInfo(name="id", hidden = true)
public final class IdCommand extends Command {
    @Override
    public void onCommand(MessageContext MC, String[] args) {
        MC.sendFormattedMessage("ID: %s", MC.getUC().getUser().getId());
    }
}

File: src/main/kotlin/moe/giga/discord/commands/IdCommand.kt
package moe.giga.discord.commands

import moe.giga.discord.contexts.MessageContext

@Suppress("unused")
class IdCommand : Command {
    override val name = "id"
    override val hidden = true

    override fun execute(MC: MessageContext, args: List<String>) {
        MC.sendMessage("ID: ${MC.userCtx.user.id}").queue()
    }
}
